app/scripts/controllers/swaps.test.js

```diff
@@ -733,14 +733,14 @@ describe('SwapsController', function () {
           gasEstimateWithRefund: '0xb8cae',
           savings: {
             fee: '-0.061067',
-            metaMaskFee: '0.5050505050505050505',
+            metaMaskFee: '0.5050505050505050505050505050505',
```

Comment:
the `swap-controller` uses the `Numiric` module in the `fetchAndSetQuotes` function, which is why we get more digits here ( that's why the swap-controller was impacted ).
I wanted to know if I should introduce the decimal as a parameter of the `Numeric` and create separated instances with specific config for each one in order to exclude the impact on the swap controller, and for that I want your opinion and suggestions first.

Reply:
@dan437 you are probably best to answer that one.

Reply:
hey @legobeat ,
thnx for your answer.
it's already computed in the UI in order to get lower precision.
i posted my comment here to mention  that  this will affect other module ( like swap-controller ) and the only way to avoid it is to use another bigNumber instance et use it exclusively in the `Numeric` package like you said.

but a question come to my mind , is that bad to have more precision ? i think a global change in the bigNumber module can be a good solution but i wanted to have other suggestions first.

Reply:
@legobeat i removed the draft , the conclusion was to add more precision and put max decimals to 36 ( we have the same limit in the core compoment )